installer/conf/rsyslog-lad.conf

```diff
@@ -0,0 +1,2 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
```

Comment:
No separate logic needed. It's already done in the existing configure_syslog.sh [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/configure_syslog.sh#L50), and that's why chose the name.